srtcore/sync.cpp

```diff
@@ -634,9 +644,12 @@ void srt::sync::CThread::create(void *(*start_routine) (void *), void *arg)
     }
 }
 
+#endif // !defined(USE_STDCXX_CHRONO)
+
 #ifdef USE_STDCXX_CHRONO
-template< class Function >
-bool srt::sync::StartThread(CThread& th, Function&& f, void* args, const char* name)
+typedef void* (&Function) (void*);
```

Comment:
`Function` isn't the best name provided this is a part of public interface. Also, all names here should be in `srt::sync` namespace.

Reply:
Renamed `Function` to `ThreadFunc` and placed inside the namespace.